### PR TITLE
feat: Add Support for HSMv2 Clusters and Backups

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: plugins/source/aws/codegen/recipes/cloudhsmv2.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+			Multiplex:  `client.ServiceAccountRegionMultiplexer("cloudhsmv2")`,
+			ExtraColumns: append(
+				defaultRegionalColumns,
+				[]codegen.ColumnDefinition{
+					{
+						Name:     "arn",
+						Type:     schema.TypeString,
+						Resolver: `resolveClusterArn`,
+						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+					},
```
Comment (hermanschaaf) [inline plugins/source/aws/codegen/recipes/cloudhsmv2.go:23]: For consistency with other AWS resources, let's add a `ResolveTagField` resolver for `tags` here (using field `TagList`)

File: plugins/source/aws/codegen/recipes/cloudhsmv2.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+			ExtraColumns: append(
+				defaultRegionalColumns,
+				[]codegen.ColumnDefinition{
+					{
+						Name:     "arn",
+						Type:     schema.TypeString,
+						Resolver: `resolveBackupArn`,
+						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+					},
+				}...),
```
Comment (hermanschaaf) [inline plugins/source/aws/codegen/recipes/cloudhsmv2.go:39]: Same comment here: for consistency with other AWS resources, let's add a `ResolveTagField` resolver for `tags` (using field `TagList`)

**Review by hermanschaaf — CHANGES_REQUESTED** [review]
LGTM, just that one thing about `tags` I think we should address

File: plugins/source/aws/codegen/recipes/cloudhsmv2.go (hunk 2 above)
Comment (bbernays) [reply plugins/source/aws/codegen/recipes/cloudhsmv2.go:50]: Done

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**